plugins/lang/js/dataset_js.cc

```diff
@@ -26,7 +26,8 @@ v8::Handle<v8::Object>
 DatasetJS::
 create(std::shared_ptr<Dataset> dataset, JsPluginContext * context)
 {
-    auto obj = context->Dataset->GetFunction()->NewInstance();
+    v8::Isolate* isolate = v8::Isolate::GetCurrent();
```

Comment:
Where is the isolate freed?  It should never be freed.  It is like that because Google's guidelines never have non-const references, and so it's a pointer.